site/assets/js/snippets.js

```diff
@@ -0,0 +1,117 @@
+// NOTICE!! DO NOT USE ANY OF THIS JAVASCRIPT
```

Comment:
Is there any way to have this working with Stackblitz ? (import the whole file each time it's needed)

Reply:
Discussed in https://github.com/twbs/bootstrap/pull/36127.